devel/qbs/Portfile

```diff
@@ -6,8 +6,8 @@ PortGroup           cxx11 1.1
 
 name                qbs
 
-version             1.8.1
-revision            2
+version             1.9.0
+revision            1
```

Comment:
`revision` starts from 0, you should set it to 0 or remove this line.

Reply:
Done